packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/XDD/Document.java

```diff
@@ -0,0 +1,60 @@
+package software.uncharted.terarium.hmiserver.models.xdd;
+
+import java.util.HashMap;
+import java.util.Hashtable;
+import java.util.List;
+
+import javax.json.bind.annotation.JsonbProperty;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+/**
+ * XDD Document representation
+ */
+public class Document {
```

Comment:
can you make this a lombok data type by adding 

`@Data
@Accessors(chain = true)`

to the class and make implement Serializable

Reply:
In my mind, we will not, ever need, to serialize XDD stuff; we are purely interested in deserializing it, so we the added annotation overhead?

Reply:
again, consistency

Reply:
done